src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -180,7 +180,8 @@ private FirefoxOptions buildFirefoxOptions(TestName testName) throws IOException
         // Config screen with many plugins can cause FF to complain JS takes too long to complete - set longer timeout
         firefoxOptions.addPreference(DOM_MAX_SCRIPT_RUN_TIME, (int)getElasticTime().seconds(600));
         firefoxOptions.addPreference(DOM_MAX_CHROME_SCRIPT_RUN_TIME, (int)getElasticTime().seconds(600));
-        firefoxOptions.addPreference(DOM_DISABLE_BEFOREUNLOAD, false);
+        firefoxOptions.addPreference(DOM_DISABLE_BEFOREUNLOAD, false); // TODO remove when we require Firefox 129 or newer
+        firefoxOptions.enableBiDi();
```

Comment:
As suggested in https://github.com/mozilla/geckodriver/issues/2182#issuecomment-2180048632.